crates/loam-cli/src/commands/build/build_clients.rs

```diff
@@ -0,0 +1,229 @@
+#![allow(clippy::struct_excessive_bools)]
+use crate::commands::build::env_toml;
+use clap::Parser;
+use soroban_cli::commands::NetworkRunnable;
+use soroban_cli::{commands as cli, CommandParser};
+use std::collections::BTreeMap as Map;
+use std::fmt::Debug;
+
+#[derive(Clone, Copy, Debug, Eq, Hash, PartialEq, clap::ValueEnum)]
+pub enum LoamEnv {
+    Development,
+    Testing,
+    Staging,
+    Production,
+}
+
+#[derive(Parser, Debug, Clone)]
+pub struct Cmd {
```

Comment:
Is this actually used as a command? Perhaps you could turn this into an Args with workspace_root as an option. Then you could flatten it into the build command.